test/Microsoft.AspNet.Mvc.TagHelpers.Test/Internal/GlobbingUrlBuilderTest.cs

```diff
@@ -348,8 +371,8 @@ public void TrimsOnlySingleLeadingSlashFromPatterns(string leadingSlash)
                 excludePattern: $"{leadingSlashes}**/*.min.css");
 
             // Assert
-            Assert.Collection(includePatterns, pattern => Assert.Equal($"{leadingSlash}**/*.css", pattern));
-            Assert.Collection(excludePatterns, pattern => Assert.Equal($"{leadingSlash}**/*.min.css", pattern));
+            Assert.Collection(includePatterns, pattern => Assert.Equal($"{prefix}**/*.css", pattern));
+            Assert.Collection(excludePatterns, pattern => Assert.Equal($"{prefix}**/*.min.css", pattern));
         }
```

Comment:
Should add a variation that has whitespace in front; since you trim the valid whitespace chars.